### PR TITLE
Update INSTALL.md and README.md

### PR DESCRIPTION
**Pull request by Shailin**
|PR checklist|Status|
|-----|:------:|
|Is issue created in GitHub?|[Yes](https://github.com/maqsoftware/Simple-Class/issues/162)|
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|NA|
|Is code analysis task executed and issues resolved in the CPP/native code?|NA|
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|NA|
|Is code commenting done properly for the changes made?|NA|
|Is correct variable naming convention followed?|NA|
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|NA|
|Are the message strings used in the application taken from strings.xml file?|NA|
|Are the permissions which are not required removed from AndroidManifest.java file?|NA|
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA|
|Are the generated files correctly configured in .gitignore?|NA|
|Are the release signing configurations removed from the gradle files?|NA|
|Is commented code removed from all files?|NA|
|Are updated images compressed (< 100 KB)?|NA|
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|Yes|

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -25,23 +20,36 @@ The steps for building the Chimple application on a Windows 7 or newer are liste
...
+## Installation steps for all apps
 1. Make sure an Android device is connected with USB debug permissions set.
 2. Overwrite the directory *tools* in ANDROID_SDK_ROOT with C:\tools
 3. `cd goa`
 4. `cocos compile -p android`
 5. Copy the content of _proj.android\libs_ folder into _proj.android-studio\goa\src\main\jniLibs_ folder. __Note__: Copy only _.so_ files.
 6. Start Android Studio and import _proj.android-studio_ folder into it.
-7. Build and run the application.
+7. Download the OBB file from the latest GitHub release and follow the instructions provided there.
+8. To change the app locale just select the desired language from the build variant.
```
Comment (alankrutap01) [inline INSTALL.md]: Add comma after locale

File: INSTALL.md [hunk 2]
```diff
@@ -25,23 +20,36 @@ The steps for building the Chimple application on a Windows 7 or newer are liste
...
+9. Set the `localeCode` in [LevelHelpScene.h](https://github.com/maqsoftware/Simple-Class/blob/master/goa/Classes/menu/LevelHelpScene.h) file as per the build variant.
+10. Build the project.
 
 ## Activate/Deactivate Kiosk Mode
 By default, the Kiosk mode is enabled. So all games are open. To disable Kiosk mode, set the KIOSK variable in
```
Comment (alankrutap01) [inline INSTALL.md]: All 'the' games

File: README.md [hunk 3]
```diff
@@ -1,25 +1,26 @@
-THIS IS THE HINDI LOCALIZATION OF CHIMPLE
-Chimple is a free, award-winning learning app. Designed for ages 3-10, Chimple engages students with 80 different learning activities, including interactive games, stories, and puzzles.
-Lessons specifically designed as games teach children reading and writing, math, science, puzzles, and coding while keeping learning fun. Level completion and bonus badges gamify each step of the learning process, ensuring advances in learning are accompanied by feelings of achievement. Chimple is always free—there are no ads or subscriptions. Go to http://chimple.org/ for more information.
+THIS IS THE HINDI LOCALIZATION OF Simple Class.
+Simple Class is a free, award-winning learning app. Designed for ages 3-10, Simple Class engages students with 80 different learning activities, including interactive games, stories, and puzzles.
```
Comment (alankrutap01) [inline README.md]: Designed for children of ages 3-10 years. Remove comma after stories

File: INSTALL.md [hunk 4]
```diff
@@ -25,23 +20,36 @@ The steps for building the Chimple application on a Windows 7 or newer are liste
...
 
-``static const bool KIOSK = false;``
+```cpp
+static const bool KIOSK = false;
+```
 
 ## FAQs
 1. How can I resolve an error encountered after running the "setup.py" script while installing cocos?
 
-    > To resolve an error encountered after running the “setup.py” script while installing cocos, check to see whether you have Python 3 installed. If you do have Python 3 installed, uninstall it and try running setup.py again.
```
Comment (alankrutap01) [inline INSTALL.md:47]: check whether you have Python 3 installed

File: INSTALL.md [hunk 5]
```diff
@@ -25,23 +20,36 @@ The steps for building the Chimple application on a Windows 7 or newer are liste
...
\ No newline at end of file
+    > To resolve an error encountered after running the “setup.py” script while installing cocos, check to see whether you have Python 3 installed. If you do have Python 3 installed, uninstall it and try running setup.py again.
+
+2. The application throws _keystore_ file not found error. How do I fix this?
+	> _keystore_ file is used to digitally sign an Android application and hence, it is not provided with the project. One must create their own _keystore_ file using the [Android KeyTool](https://developer.android.com/studio/publish/app-signing) which comes with the Android Studio itself in order to sign the APK.
```
Comment (alankrutap01) [inline INSTALL.md]: Remove 'itself' after 'Android Studio'

File: README.md [hunk 6]
```diff
@@ -1,25 +1,26 @@
...
+Lessons specifically designed as games teach children reading and writing, math, science, puzzles, and coding while keeping learning fun. Level completion and bonus badges gamify each step of the learning process, ensuring advances in learning are accompanied by feelings of achievement. Simple Class is always free—there are no ads or subscriptions. Go to http://chimple.org/ for more information.
```
Comment (alankrutap01) [inline README.md]: keeping the learning process fun-filled.
Accompanied by 'the' feelings

File: README.md [hunk 7]
```diff
@@ -1,25 +1,26 @@
...
 
-Chimple helps kids:
+Simple Class helps kids:
 Learn on their own: Kids learn using 80 fun different activities in a variety of subject areas.
```
Comment (alankrutap01) [inline README.md]: 80 different fun activities

File: README.md [hunk 8]
```diff
@@ -1,25 +1,26 @@
...
 Stay interested: Kids learn through games, tying early learning to achievement and not work.
 Personalize instructions: Kids choose from a variety of lessons in reading and writing, math, science, puzzles, and coding. The app includes 13 alphabet sessions, 5 writing sessions, 19 sentences sessions,20 numbers sessions, 15 stories, and 8 map activities.
-Read without distractions: Chimple works offline, without ads, allowing children to focus on the current lesson.
+Read without distractions: Simple Class works offline, without ads, allowing children to focus on the current lesson.
 Explore the world around them: The app’s stories cover a variety of engaging topics, including everyday life, animals, friendship, and sports.
 
-AWARD-WINNING APP
+AWARD-WINNING APP:
 Chimple was a finalist in the $15 million Global Learning XPRIZE based on improvement in children’s learning outcomes and ease of use.
```
Comment (alankrutap01) [inline README.md]: Change Chimple to Simple Class

File: INSTALL.md [hunk 9]
```diff
@@ -25,23 +18,36 @@ The steps for building the Chimple application on a Windows 7 or newer are liste
...
     2. NDK_ROOT: C:\ndk\android-ndk-r12b
     3. ANT_ROOT: C:\ant\apache-ant-x.xx.x\bin
-## Build and install goa
-Goa is an application which has many games for a child to play while improving literacy.
+## Installation steps for all apps
 1. Make sure an Android device is connected with USB debug permissions set.
 2. Overwrite the directory *tools* in ANDROID_SDK_ROOT with C:\tools
 3. `cd goa`
 4. `cocos compile -p android`
 5. Copy the content of _proj.android\libs_ folder into _proj.android-studio\goa\src\main\jniLibs_ folder. __Note__: Copy only _.so_ files.
```
Comment (ankitk20) [inline INSTALL.md]: Update this to copy to the flavor folder

File: INSTALL.md [hunk 10]
```diff
@@ -25,23 +18,36 @@ The steps for building the Chimple application on a Windows 7 or newer are liste
...
 6. Start Android Studio and import _proj.android-studio_ folder into it.
-7. Build and run the application.
+7. Download the OBB file from the latest GitHub release and follow the instructions provided there.
```
Comment (ankitk20) [inline INSTALL.md]: Hyperlink this to point to [GitHub release](github.com/maqsoftware/Simple-Class/releases)

File: README.md [hunk 11]
```diff
@@ -1,25 +1,26 @@
-THIS IS THE HINDI LOCALIZATION OF CHIMPLE
-Chimple is a free, award-winning learning app. Designed for ages 3-10, Chimple engages students with 80 different learning activities, including interactive games, stories, and puzzles.
-Lessons specifically designed as games teach children reading and writing, math, science, puzzles, and coding while keeping learning fun. Level completion and bonus badges gamify each step of the learning process, ensuring advances in learning are accompanied by feelings of achievement. Chimple is always free—there are no ads or subscriptions. Go to http://chimple.org/ for more information.
+THIS IS THE HINDI LOCALIZATION OF Simple Class.
+Simple Class is a free, award-winning learning app. Designed for children of ages 3-10 years, Simple Class engages students with 80 different learning activities, including interactive games, stories and puzzles.
```
Comment (ankitk20) [inline README.md]: Update README referring to latest description sent by Alex

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: INSTALL.md [hunk 12]
```diff
@@ -25,23 +18,36 @@ The steps for building the Chimple application on a Windows 7 or newer are liste
     1. ANDROID_SDK_ROOT: Enter the directory from Step 1.4 above
     2. NDK_ROOT: C:\ndk\android-ndk-r12b
     3. ANT_ROOT: C:\ant\apache-ant-x.xx.x\bin
```
Comment (ankitk20) [inline INSTALL.md:20]: Add one step for copying Firebase library

**Review by ankitk20 — DISMISSED** [review]

File: INSTALL.md (hunk 4 above)
Comment (mathewnik90) [reply INSTALL.md:47]: I think she mentioned to remove to see in the previous sentence. The current construction of the second sentence is not correct.

File: README.md [hunk 13]
```diff
@@ -1,25 +1,30 @@
...
+**Stay interested**: Kids learn through games, tying early learning to achievement and not work.
+
+**Personalize instructions**: Kids choose from a variety of lessons in reading and writing, math, science, puzzles, and coding. The app includes 13 alphabet sessions, 5 writing sessions, 19 sentences sessions,20 numbers sessions, 15 stories, and 8 map activities.
+
+**Read without distractions**: Simple Class works offline, without ads, allowing children to focus on the current lesson.
+
+**Explore the world around them**: The app’s stories cover a variety of engaging topics, including everyday life, animals, friendship, and sports.
+
+**ONE TIME SET UP INSTRUCTIONS:**
+1. Download APK to your local computer.
```
Comment (mathewnik90) [inline README.md]: Download the APK

**Review by mathewnik90 — CHANGES_REQUESTED** [review]

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**